pages/_app.js

```diff
@@ -1,14 +1,3 @@
-import GlobalStyle from "@/styles";
-import Head from "next/head";
-
-export default function App({ Component, pageProps }) {
-  return (
-    <>
-      <GlobalStyle />
-      <Head>
-        <title>Capstone Project</title>
-      </Head>
-      <Component {...pageProps} />
-    </>
-  );
+export default function App({ Component, pageProps: { pageProps } }) {
```

Comment:
i don't know for what reason you removed global styles . but maybe you have your reasons for that but in general it is better to use globalStyles for the app

1.Consistency: By defining your styles in one central location, you can ensure that the same styles are applied across all pages of your application.

2.Performance: By defining your styles globally, you avoid the overhead of loading and parsing CSS files for each individual page, which can improve the overall performance of your application.

3.Overriding Third-party Styles: If you are using any third-party components or libraries in your application, global styles defined in _app.js will take precedence over any styles defined by those components, allowing you to more easily override and customize their appearance.

4.Easy Maintenance: Centralizing your styles in one place makes it easier to manage and maintain your application over time, since you only need to make changes in one location to affect the entire application.